docs/data/sdks/ios-swift/index.md

```diff
@@ -0,0 +1,484 @@
+---
+title: iOS SDK
+description: The Amplitude iOS SDK installation and quick start guide. 
+icon: simple/ios
+---
+
+
+![CocoaPods](https://img.shields.io/cocoapods/v/AmplitudeSwift)
+
+This is the official documentation for the Amplitude Analytics iOS SDK.
+
+!!!info "iOS Swift SDK Resources (Beta)"
+    [:material-github: GitHub](https://github.com/amplitude/Amplitude-Swift) · [:material-code-tags-check: Releases](https://github.com/amplitude/Amplitude-Swift/releases)
+
+--8<-- "includes/no-ampli.md"
+    To use Ampli see the [non-Beta SDK](../../sdks/ios/) and [Ampli Wrapper](../../sdks/ios/ampli/) instead.
+
+## Getting started
+
+### Install
+
+Install the Amplitude Analytics iOS SDK via CocoaPods, Carthage, or Swift Package Manager.
+
+=== "CocoaPods"
+
+    1. Add dependency to `Podfile`.
+
+        ```bash
+        pod 'AmplitudeSwift', '~> 0.3.0'
+        ```
+
+    2. Run `pod install` in the project directory to download dependency.
+
+=== "Swift Package Manager"
+
+    1. Navigate to `File` > `Swift Package Manager` > `Add Package Dependency`.
+    2. Enter `https://github.com/amplitude/Amplitude-Swift` into the search bar.
+    3. It automatically resolves to the latest version.
+
+    The Amplitude-Swift package appears as a dependency after it's added.
+
+=== "Carthage"
+
+    Add the following line to your `Cartfile`.
+      
+    ```bash
+    github "amplitude/Amplitude-Swift" ~> 0.3.0
+    ```
+
+
+### Initialization
+
+Before you can instrument, you must initialize the SDK using the API key for your Amplitude project. You can use the iOS SDK anywhere after it's initialized in an iOS application.
+
+```swift
```

Comment:
Ideally we can have obj-c as well, I am fine to start with swift now

Reply:
Agree with the obj-c examples, I will create a ticket to track it, and start with swift for now.